internal/operators/ocs/validations.go

```diff
@@ -197,7 +196,7 @@ func validateRequirementsForMode(o *operator, nodeInfo *resourceInfo, mode ocsDe
 		totalRAM = o.config.OCSRequiredCompactModeRAMGB
 	} else if mode == minimalMode {
 		totalCPUs = o.config.OCSMinimumCPUCount
-		totalRAM = o.config.OCSMinimumRAMGB
+		totalRAM = o.config.OCSRequiredRAMGB // RAM requirements for minimal and standard mode are same
```

Comment:
I don't think this comment is necessary.

Reply:
done